From a7d97b966993214a5ea8928aec6ed5fa6e32f939 Mon Sep 17 00:00:00 2001
From: Vadim Zeitlin <vadim@wxwidgets.org>
Date: Tue, 18 Mar 2025 14:48:15 +0100
Subject: [PATCH] Don't use C++11 member initialization in wxToastEventHandler

This fixes MSVS 2010 build after the changes of 669438636b (Fix events
generated when toast-based notifications are dismissed, 2024-09-23).
---
 src/msw/rt/notifmsgrt.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- src/msw/rt/notifmsgrt.cpp
+++ src/msw/rt/notifmsgrt.cpp
@@ -51,7 +51,7 @@ class wxToastEventHandler :
     wxToastEventHandler(wxToastNotifMsgImpl* toastImpl) :
         m_impl(toastImpl)
     {
-
+        m_cRef = 0;
     }
 
     void Detach()
@@ -115,7 +115,7 @@ class wxToastEventHandler :
     }
 
 private:
-    ULONG m_cRef = 0;
+    ULONG m_cRef;
 
     wxToastNotifMsgImpl* m_impl;
 };
